tla/consensus/MCabs.cfg

```diff
@@ -6,14 +6,23 @@ CONSTANTS
     NodeThree = n3
     Servers <- MCServers
     Terms <- MCTerms
-    MaxLogLength <- MCMaxLogLength
+    Seq <- MCSeq
+    TypeOK <- MCTypeOK
 
 INVARIANTS
     NoConflicts
     TypeOK
 
 PROPERTIES 
     AppendOnlyProp
+    \* EquivExtendProp
```

Comment:
Why are those commented out?

Reply:
I assume they are not checked by default for performance reasons (although I'm not sure as abs is fairly quick to check compared to ccfraft

Reply:
> Finished in 03s at (2024-10-01 15:05:45)

I'd rather have them enabled.

Reply:
Checking `EquivExtendProp` and `EquivCopyMaxAndExtendProp` during "ordinary" model checking of `abs` is expensive, and I don't think we want to pay the overhead for as long as the definitions of the actions are simple.  For now, we might as well disable model checking of `abs`, but  I expect that we will add more detail soon.

Reply:
> For now, we might as well disable model checking of abs, but I expect that we will add more detail soon.

Let's discuss that.